test/Parse/environment.swift

```diff
@@ -0,0 +1,51 @@
+// RUN: %swift_plain -target i686-unknown-windows-cygnus -parse-as-library -parse-stdlib %s -emit-ir -o - | FileCheck %s -check-prefix CHECK-CYGWIN
+// RUN: %swift_plain -target i686-unknown-windows-gnu -parse-as-library -parse-stdlib %s -emit-ir -o - | FileCheck %s -check-prefix CHECK-MINGW
+// RUN: %swift_plain -target i686-unknown-windows-itanium -parse-as-library -parse-stdlib %s -emit-ir -o - | FileCheck %s -check-prefix CHECK-ITANIUM
+// RUN: %swift_plain -target i686-unknown-windows-msvc -parse-as-library -parse-stdlib %s -emit-ir -o - | FileCheck %s -check-prefix CHECK-MSVC
+// RUN: %swift_plain -target i686-unknown-linux-gnu -parse-as-library -parse-stdlib %s -emit-ir -o - | FileCheck %s -check-prefix CHECK-LINUX
```

Comment:
I think this belongs better in `test/Parse/ConditionalCompilation`, where we have one file per triple.